src/Microsoft.Data.Analysis.Interactive/Directory.Build.targets

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <Import Project="./../../Directory.Build.targets" />
+
+  <!-- Overriding PackDependsOn so instead we just call Pack on the MDA project. -->
+  <PropertyGroup>
+    <PackDependsOn>PackMDAProject</PackDependsOn>
```

Comment:
Does this conflict with having `<IsPackable>false</IsPackable>` in the .csproj?

I don't think we need this. `dotnet pack` on this project is fine to be a no-op.

Reply:
> Does this conflict with having <IsPackable>false</IsPackable> in the .csproj?

Nope, IsPackable only matters when you actually use the Pack targets in the SDK but the fact that we are overriding this here makes it that setting it to false is actually not required any longer.

> I don't think we need this. dotnet pack on this project is fine to be a no-op.

I'm fine with removing this file too, I only added it in case people wanted to be able to call dotnet pack on this project and actually generate the package where this assembly is packed, even if it is not part of this project. I'm fine with either keeping it or removing it in case we don't think a lot of people will use it.